relay-event-schema/src/protocol/span.rs

```diff
@@ -134,9 +134,8 @@ pub struct Span {
     )]
     pub performance_issues_spans: Annotated<bool>,
 
-    // TODO remove retain when the api stabilizes
     /// Additional arbitrary fields for forwards compatibility.
-    #[metastructure(additional_properties, retain = true, pii = "maybe")]
+    #[metastructure(additional_properties, pii = "maybe")]
```

Comment:
With this change, the bad `end_timestamp` would have been removed before converting to span V2.